WordPress/Classes/Utility/Media/ImageRequest.swift

```diff
@@ -28,7 +28,7 @@ final class ImageRequest {
 }
 
 struct ImageRequestOptions {
-    /// Resize the thumbnail to the given size. By default, `nil`.
+    /// Resize the thumbnail to the given size (in pixels). By default, `nil`.
     var size: CGSize?
```

Comment:
I feel like I have commented about this before... I apologize if you have already answered this question... 😅 

I think it's good to express "the given size (in pixels)" in code. Like `sizeInPixel: CGSize`, or creating a `struct PixelSize { pixelWidth, pixelHeight }` type.

It's easy to miss documentation, but it's very hard to miss if you have to write "pixel" in code.